.stylelintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "stylelint-config-standard",
+	"rules": {
+		"color-hex-case": "upper",
```

Comment:
The convention in the project was mixed so i wasnt sure what to choose. I'll fix this